### PR TITLE
[APPLS-1980] Add DR Custom Components package to streamlit base env

### PR DESCRIPTION
**Pull request by kideh88**
## Summary
Updates the base environment `[Experimental] Python 3.9 Streamlit` 
- Add DR custom components package for Streamlit (v0.1.3)
- Update streamlit version to 1.31.0
- Add streamlit_extras lib version 0.3.6

## Rationale
Preparation for upcoming functionality in Streamlit apps. Documentation for DR custom components is on the way

### CODE REVIEW

File: public_custom_application_environments/python39_streamlit/Dockerfile [hunk 1]
```diff
@@ -9,8 +9,10 @@ RUN apt-get update && apt-get install -y \
     git \
     && rm -rf /var/lib/apt/lists/*
 
+COPY ./packages/dr_components-0.1.2-py3-none-any.whl /app/
```
Comment (demchukilya) [inline public_custom_application_environments/python39_streamlit/Dockerfile:12]: It looks like you haven't added `packages` folder to repo.

**Review by demchukilya — CHANGES_REQUESTED** [review]
It looks like part of changes is missing.

File: public_custom_application_environments/python39_streamlit/Dockerfile (hunk 1 above)
Comment (kideh88) [reply public_custom_application_environments/python39_streamlit/Dockerfile:12]: oh, thanks for catching! :face_in_clouds:  Fixed now

**Review by demchukilya — APPROVED** [review]
LGTM

**Comment by rvorobii** [comment]
@yakov-g could you review this one please?

**Pull request merged**